Gordon360/Controllers/RecIM/ActivitiesController.cs

```diff
@@ -21,22 +21,22 @@ public ActivitiesController(IActivityService activityService)
         }
 
         ///<summary>Gets a list of all Activities by parameter </summary>
-        ///<param name="active"> Optional active parameter </param>
-        ///<param name="time"> Optional time parameter </param>
+        ///<param name="active"> Optional active parameter denoting whether or not an activity has been completed </param>
         /// <returns>
         /// All Existing Activities 
         /// </returns>
         [HttpGet]
         [Route("")]
-        public ActionResult<IEnumerable<ActivityExtendedViewModel>> GetActivities([FromQuery] DateTime? time, bool active)
+        public ActionResult<IEnumerable<ActivityExtendedViewModel>> GetActivities([FromQuery] bool? active)
         {   
-            if (time is null && active)
+            if ( active is bool isActive)
```

Comment:
I think this would be clearer as a `not null` check.
```suggestion
            if ( active is not null)
```

Reply:
resolved in person